File: Electronics_Shop_17.Model/Requests/AddSeller.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddSeller
    {
        public string StoreName { get; set; }
        public int LicenseNumber { get; set; }
        public int AdressId { get; set; }
        public int UserAccountId { get; set; }
        public int PersonId { get; set; }
        public bool isDeleted { get; set; } = false;

    }
}

File: angular17_web/src/app/models/seller/add.seller.ts
export class AddSeller {
  storeName!: string;
  licenseNumber!: number;
  adressId!: number;
  userAccountId!: number;
  personId!: number;
  isDeleted?: boolean;

  constructor(data?: any) {
    if (!data) return;

    this.storeName = data.storeName;
    this.licenseNumber = data.licenseNumber;
    this.adressId = data.adressId;
    this.userAccountId = data.userAccountId;
    this.personId = data.personId;
    this.isDeleted = data.isDeleted;
  }
}
